build.gradle

```diff
@@ -122,10 +149,6 @@ task smokeTest(type: Test) {
     }
```

Comment:
We can remove the smokeTest run config as well now

Reply:
I'm also not sure about the whole WIP thing either. I think the correct way to deal with that is to use @Disabled on any tests which are in progress.